.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
   - pip install --upgrade pip
   - pip install --progress-bar off torch==$TORCH_VER
   - pip install --progress-bar off tensorflow==$TENSORFLOW_VER
+  - pip install --progress-bar off spacy==2.2.4
```

Comment:
What cause this strict dependency to spacy?

Reply:
It seems that AllenNLP may change the dependencies on their 1.0.0 version on Pip. Not sure what can cause this. Someone is working to fix this here: https://github.com/asyml/forte/pull/321